packages/react-native/tooling/posthog.gradle

```diff
@@ -157,6 +157,40 @@ plugins.withId('com.android.application') {
     }
 }
 
+def resolvePostHogCliPackagePath(reactRoot) {
+    def resolvedCliPath = null
+    try {
+        resolvedCliPath = new File(["node", "--print", "require.resolve('@posthog/cli/package.json')"].execute(null, rootDir).text.trim()).getParentFile();
+    } catch (Throwable ignored) { // Check if it's located in .pnpm
+        try {
+            def pnpmRefPath = reactRoot.toString() + "/node_modules/@posthog/cli/node_modules/.bin/posthog-cli"
+            def postHogCliFile = new File(pnpmRefPath)
+
+            if (postHogCliFile.exists()) {
+                def cliFileText = postHogCliFile.text
+                def matcher = cliFileText =~ /NODE_PATH="([^"]*?)@posthog\/cli\//
+
+                if (matcher.find()) {
+                    def match = matcher.group(1)
+                    resolvedCliPath = new File(match + "@posthog/cli")
+                }
+            }
+        } catch (Throwable ignored2) {} // if the resolve fails we fallback to the default path
+    }
+
+    def cliPackage = null
+
+    if (resolvedCliPath != null && resolvedCliPath.exists()) {
+        cliPackage = resolvedCliPath.getAbsolutePath()
+    } else if (new File("$reactRoot/node_modules/@posthog/cli").exists()) {
+        cliPackage = "$reactRoot/node_modules/@posthog/cli"
```

Comment:
do I need to append `run-posthog-cli.js` to this file path for being able to call `./node_modules/@posthog/cli/run-posthog-cli.js exp ...`?